integration/repo_test.go

```diff
@@ -360,6 +369,15 @@ func TestRepository(t *testing.T) {
 			},
 			golden: "testdata/conda-spdx.json.golden",
 		},
+		{
+			name: "conda environment.yaml generating SPDX SBOM",
+			args: args{
+				command: "fs",
+				format:  "spdx-json",
+				input:   "testdata/fixtures/repo/conda-environment",
+			},
+			golden: "testdata/conda-environment-spdx.json.golden",
+		},
```

Comment:
Since we already have an intermediate SBOM representation, I think either CycloneDX or SPDX is enough.

Reply:
Removed SPDX in https://github.com/aquasecurity/trivy/pull/6569/commits/d5b0a5693f3f02e6b7a80f10cbee725b3e1a12b3